code/modules/antagonists/incursion/incursion.dm

```diff
@@ -142,14 +142,17 @@
 
 	var/purchases = ""
 	var/TC_uses = 0
+	var/effective_tc = 0
 	LAZYINITLIST(GLOB.uplink_purchase_logs_by_key)
 	for(var/I in members)
 		var/datum/mind/syndicate = I
 		var/datum/uplink_purchase_log/H = GLOB.uplink_purchase_logs_by_key[syndicate.key]
 		if(H)
 			TC_uses += H.total_spent
+			effective_tc += H.effective_amount
 			purchases += H.generate_render(show_key = FALSE)
-	parts += "(Syndicates used [TC_uses] TC) [purchases]"
+	var/effective_message = TC_uses < effective_tc ? " / effectively worth with [effective_tc] TC" : ""
```

Comment:
maybe im having a pea brained ESL moment, but i dont think there should be a "with" there and it should just be "effectively worth [amount] TC"

Reply:
I don't know what's the best wording in fact. Would it be better?

Reply:
Pretty sure yea. maybe someone else can weigh in as i am the funny ESL like i said, but im 99.5% sure that the with dosnt go there.

Reply:
Correct, the "with" should not be there.